.babelrc

```diff
@@ -0,0 +1,10 @@
+{
+  "env": {
+    "@babel/preset-env",	      "test": {
+    "@babel/preset-react"	          "presets": [
+  ]	              "@babel/preset-env",
+              "@babel/preset-react"
+          ]
+      }
+  }
+}
```

Comment:
Thanks for showing me a little bit of github. :)
You may need to get rid of 
@@ 3 @babel/preset-env,
@@ 4 @babel/preset-react